in_text/text_data_release.yml

```
@@ -228,74 +234,74 @@ entities:
   -
     data-name: 03_model/in/saltfront_updated.csv
     data-description: >-
-      XX
+      a record of the salt front location in units of river mile, provided by the Delaware River Basin Commission
     attributes: 
     -
       attr-label: datetime
       attr-def: >-
-        XX
+        date
       attr-defs: >-
         This release
       data-min: "2000-01-01"
       data-max: "2021-12-31"
-      data-units: NA
+      data-units: date YYYY-MM-DD
     -
       attr-label: saltfront_daily
       attr-def: >-
-        XX
+        Daily record of the salt front location
       attr-defs: >-
         This release
       data-min: 30.73
       data-max: 90.74
-      data-units: XX
+      data-units: river miles
     -
       attr-label: saltfront7_weekly
       attr-def: >-
-        XX
+        7-day backward looking average of the salt front location
       attr-defs: >-
         This release
       data-min: 30.73
       data-max: 89.75
-      data-units: XX
+      data-units: river miles
   -
     data-name: 03_model/in/COAWST_model_runs/processed/COAWST_****_7day.csv
```

swapping *s for {}, after discussion with Margaux - we decided this would be a better format
```suggestion
    data-name: 03_model/in/COAWST_model_runs/processed/COAWST_{year}_7day.csv
```